lib/modules/manager/html/readme.md

```diff
@@ -1 +1,7 @@
-The `html` manager extracts scripts and CSS includes pointing to cdnjs. It differs from the `cdnjs` manager in that it detects if an integrity hash is already present and updates it too if found.
+The `html` manager updates `<script>` tags and _CSS includes_ that point to the [cdnjs content delivery network](https://cdnjs.com/).
+It also updates Subresource Integrity (SRI) hashes in `integrity` attributes.
+
```

Comment:
Can one of the maintainers read the `html` code for me, and check if the docs match the code? I'm bad at reading the code. 🙈